README.md

```diff
@@ -1120,12 +1120,13 @@ present script similar to how `$0` works in bash or perl.
 
 `opts` is optional and acts like calling `.options(opts)`.
 
-.version(version, [option], [description])
+.version([option], [description], version)
 ----------------------------------------
 
 Add an option (e.g. `--version`) that displays the version number (given by the
-`version` parameter) and exits the process. If present, the `description`
-parameter customizes the description of the version option in the usage string.
+`version` parameter) and exits the process, if not provided, it will default to `--version`.
```

Comment:
I would make the last part its own sentence:

```
`version` parameter) and exits the process. If not provided, it will default to `--version`.
```